zboxcore/sdk/renameworker.go

```diff
@@ -121,7 +135,7 @@ func (req *RenameRequest) ProcessRename() error {
 		return fmt.Errorf("Rename failed: %s", err.Error())
 	}
 
-	req.consensus = 0
+	req.consensus.consensus = float32(removedNum)
```

Comment:
req.consensus should be 0.
Previous was consensus on operation. Now it needs to be calculated for commit consensus.

Reply:
Resolved